src/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Tracer } from '@aws-lambda-powertools/tracer';
+import {
+  BatchGetItemCommand,
+  BatchGetItemCommandInput,
+  BatchWriteItemCommand,
+  BatchWriteItemCommandInput,
+  DeleteItemCommand,
+  DeleteItemCommandInput,
+  DynamoDBClient,
+  DynamoDBClientConfig,
+  GetItemCommand,
+  GetItemCommandInput,
+  PutItemCommand,
+  PutItemCommandInput,
+} from '@aws-sdk/client-dynamodb';
+
+export class MhDynamoDbClient {
+  private readonly client: DynamoDBClient;
+
+  constructor(config: DynamoDBClientConfig) {
+    const tracer = new Tracer();
+    this.client = tracer.captureAWSv3Client(new DynamoDBClient(config));
+  }
+
+  async getItem<T>(args: GetItemCommandInput): Promise<T> {
+    const response = await this.client.send(new GetItemCommand(args));
```

Comment:
I believe we want to use the documentClient (high level?) and not the "raw" (low-level?) for our commands. Not sure if the interface differs?

Reply:
It does to a great degree, the low level stuff uses dynamodb json syntax (e.g. `PK.S` instead of just `PK` and we don't want that (imo)